frontend/src/Routes.tsx

```diff
@@ -3,6 +3,7 @@ import { Redirect, Route, Switch } from 'react-router-dom';
 
 import { Bullseye, Spinner } from '@patternfly/react-core';
 import EmptyRoute from '@console/mount/src/components/foundation/static-routes/EmptyRoute';
+import TestK8s from './poc-code/testK8s/TestK8s';
```

Comment:
This should be dynamically loaded, so we don't increate the size of main bundle
```suggestion
const TestK8s = React.lazy(() => import(/* webpackChunkName: "TestK8s" */'./poc-code/testK8s/TestK8s'));
```